game/src/ecs/components/ai/AITools.java

```diff
@@ -26,6 +26,10 @@ public class AITools {
      * @param path Path on which the entity moves
      */
     public static void move(Entity entity, GraphPath<Tile> path) {
+        // entity is already at the end
+        if (pathFinished(entity, path)) {
```

Comment:
in der Methode wird
` return path.get(path.getCount() - 1).equals(level.getTileAt(pc.getPosition().toCoordinate()));` gemacht. 
Ist das -1 denn richtig? Müsste ja dann eigentlich ein Feld zu früh sein, oder?